src/platform/linux/wayland/window.rs

```diff
@@ -45,12 +47,22 @@ impl Window {
             surface.clone(),
             (width, height),
             move |event| match event {
-                WEvent::Configure { new_size, .. } => {
+                WEvent::Configure { new_size, states } => {
                     let mut store = window_store.lock().unwrap();
+
+                    let mut is_fullscreen = false;
+                    for state in states {
+                        if let WState::Fullscreen = state {
+                            is_fullscreen = true;
+                            break;
+                        }
+                    }
```

Comment:
`WState` implements `PartialEq`, so you should be able to contract this to:

```rust
let is_fullscreen = states.contains(&WState::Fullscreen);
```

Reply:
Good point! That's much nicer. I've updated the commit with that change. :)